host_cleanup.sh

```diff
@@ -21,3 +21,6 @@ ANSIBLE_FORCE_COLOR=true ansible-playbook \
 sudo rm -rf /etc/NetworkManager/dnsmasq.d/openshift.conf /etc/NetworkManager/conf.d/dnsmasq.conf
 sudo virsh net-destroy baremetal
 sudo virsh net-undefine baremetal
+
+# Cleanup ssh keys for baremetal network
+sed -i "/^192.168.111/d" /home/$USER/.ssh/known_hosts
```

Comment:
I think I'd get more value from this if we put it into "ocp_cleanup.sh", as I regularly run it to bring up a new bootstrap node without running host_cleanup.sh, happy to merge it here either if you think its a better fit.

Reply:
Yeah that could work too - particularly if we included stopping the openshift VMs in ocp_cleanup, e.g the undeploy part we do when re-running 07?

Reply:
Ya, we could add that too. I guess it would be cleaner that way. I usually just rerun 07_.. as the undeploy in there covers me anyways.